api/src/main/resources/workbench.yaml

```diff
@@ -1799,3 +2021,196 @@ definitions:
+        type: integer
+        format: int64
+      conceptName:
+        description: name of concept
+        type: string
+      domainId:
+        description: domain of concept
+        type: string
+      vocabularyId:
+        description: vocabulary of concept
+        type: string
+      conceptCode:
+        description: original vocab code of concept
+        type: string
+      conceptClassId:
+        description: class of concept
+        type: string
+      standardConcept:
+        description: standard concept value 1 char
+        type: string
+      countValue:
+        description: est count in the cdr
+        type: integer
+        format: int64
+      prevalence:
+        description: prevalence among participants percent count divided num participants
+        type: number
+        format: float
+
+
+  ConceptListResponse:
+      type: object
+      required:
+        - items
+      properties:
+        items:
+          type: "array"
+          items:
+            $ref: "#/definitions/Concept"
+
+  AnalysisResult:
+      type: object
+      required:
+        - analysisId
+
+      properties:
+        id:
+          description: pk id for jpa code to work
+          type: integer
+          format: int64
+        analysisId:
+          description: id analysis
+          type: integer
+          format: int64
+        countValue:
+          description: count
+          type: integer
+          format: int64
+        stratum1:
+          description: stratum 1
+          type: string
+        stratum1Name:
+          description:  usually concept name corresponding to stratum
+          type: string
+        stratum2:
+          description: stratum 2
+          type: string
+        stratum2Name:
+          description:  usually concept name corresponding to stratum
+          type: string
+        stratum3:
+          description: stratum 3
+          type: string
+        stratum3Name:
+          description:  usually concept name corresponding to stratum
+          type: string
+        stratum4:
+          description: stratum 4
+          type: string
+        stratum4Name:
+          description:  usually concept name corresponding to stratum
+          type: string
+        stratum5:
+          description: stratum 5
+          type: string
+        stratum5Name:
+          description:  usually concept name corresponding to stratum
+          type: string
+
+
+  AnalysisResultListResponse:
+      type: object
+      required:
+        - items
+      properties:
+        items:
+          type: "array"
+          items:
+            $ref: "#/definitions/AnalysisResult"
+
+
+
+  Analysis:
+      type: object
+      required:
+        - analysisId
+      properties:
+        analysisId:
+          description: id analysis
+          type: integer
+          format: int64
+        analysisName:
+          description:  analysis name
+          type: string
+        stratum1Name:
```

Comment:
As mentioned previously: I don't think we should be exposing the Achilles schema in our REST API in this manner. We should make REST APIs that are tailored to the data we're returning as much as possible... generic representations like this are hard to use for clients, and tie us to a particular implementation.

(I won't hold up this review on it, but something for us to consider changing in future.)

Reply:
Ok. thanks. I mean the analysis is a concept that the front end classes use for charting.   It's

Reply:
It is the list of things we can chart in our front end.  I thought it was as good an abstract concept as any for the front end so i just use it.